src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.forms {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input_big {
+  font-family: inherit;
```

Comment:
create common style for two searchbars and move all repetitive  styles there to prevent duplicating